docs/program-setup/serial/serial_communication.md

```diff
@@ -30,12 +52,9 @@ mbed sterm
 
 `mbed sterm` starts message printing.
 
-Restart the application using the board's reset button, or by entering <kbd>Ctrl + B</kbd> in the serial terminal.
-The example prints "Hello World!" to the terminal after the reset.
```

Comment:
I've removed this because now we introduce `mbed compile --flash` method first, which doesn't require a reset to see "Hello World!" in our example.